src/client/components/BaselayerControl.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
I would expect this component to `emit` something so that a smart component can pick it up and do the actual work.

Reply:
it's a component that gets rendered onto the map, so emitting won't really give us any benefits.